### PR TITLE
WT-6842 Add example showing using src_id only.

### PR DESCRIPTION
**Pull request by sueloverso**
(And a typo fix in another file.)

This functionality already worked so I just added a pass in the example program to use it.

### CODE REVIEW

File: examples/c/ex_backup_block.c [hunk 1]
```diff
@@ -328,12 +365,20 @@ take_incr_backup(WT_SESSION *session, int i)
...
     char buf[1024], h[256], *tmp;
     const char *filename;
-    bool first;
+    bool first, src_only;
 
     tmp = NULL;
     tmp_sz = 0;
     /* Open the backup data source for incremental backup. */
-    (void)snprintf(buf, sizeof(buf), "incremental=(src_id=\"ID%d\",this_id=\"ID%d\")", i - 1, i);
+    if (i == 1) {
```
Comment (keitharnoldsmith) [inline examples/c/ex_backup_block.c:373]: `if (i == SRCONLY_ID)` ?

**Review by keitharnoldsmith — APPROVED** [review]
LGTM.  

There are a couple of minor comments below.

Also, does the documentation for Block-based Incremental Backup need to be updated?

As I'm sure you know (since you wrote it!) a lot of the special case code for the src_id only test are near-duplicates of the code for the standard incremental cases. I didn't see an obvious and simple way to refactor to reduce/simplify the duplication.

**Review by ddanderson — COMMENTED** [review]
I'm worried that the addition of this case makes the example harder to understand.  If, as I understand, the srconly approach is an alternate way of doing things, that is not clear in the example.  I think an example, (esp. for something that is medium-complicated like backup), should pick one clear way that developers can copy.  If we need test coverage for this feature, it should really be in a new test.

**Comment by sueloverso** [comment]
@ddanderson thanks for that. I agree that adding this makes the example less readable. I will look into adding a smaller test instead. I originally tried to add it into the Python test but that got messy too. I think I will just add a small stand-alone test that focuses only on this.

File: test/suite/test_backup14.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def take_full_backup(self):
                 # Take a full bakcup into each incremental directory
```
[the remark this answers is not in the document]
Comment (sueloverso) [reply test/suite/test_backup14.py]: This typo ended up in 3 files. I fixed that, and the grammar below in all 3.

**Comment by sueloverso** [comment]
I have taken one of the backup tests and trimmed it down to just use the source id as the test, and left the example program unchanged.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM

**Review by ddanderson — APPROVED** [review]
LGTM.

Possibly as a followup ticket, I'd like to see some refactoring of the backup python tests.  test_backup15.py and this new example look very similar.  Perhaps they could inherit from a base class that provldes common functions?

**Pull request merged**